test/combinations.jl

```diff
@@ -516,6 +516,53 @@ end
     @test_throws "not supported" volumeintegral(f, parab)
 end
 
+@testitem "ParametrizedCurve" setup=[Setup] begin
+    # ParametrizedCurve has been added in Meshes v0.51.20
+    # If the version is specified as minimal compat bound in the Project.toml, the downgrade test fails
+    if pkgversion(Meshes) >= v"0.51.20"
+        using CoordRefSystems: Polar
+        using LinearAlgebra: norm
+
+        radius = 4.4
```

Comment:
```suggestion
        # Parameterize a circle centered on origin with specified radius
        radius = 4.4
```

Reply:
I think adding a comment here might be useful to help us remember what this geometry is modeling.